src/utils/mathFormulas.js

```diff
@@ -0,0 +1,7 @@
+const stdev = (array) => {
+  const n = array.length;
+  const mean = array.reduce((a, b) => a + b) / n;
+  return Math.sqrt(array.map((x) => (x - mean) ** 2).reduce((a, b) => a + b) / n);
```

Comment:
same with this reduce, and there is a `Math.pow` function

Reply:
Actually to make this more readable I would also create here a separate function:
`const mean = (array) => (_.sum(array)/array.length);`
and use it in hte two lines relevant for std

Reply:
I was using `Math.pow`, but the linter complained. Changed to using `_.sum` instead of `reduce`